torsiondrive/dihedral_scanner.py

```diff
@@ -101,7 +101,7 @@ def __init__(self, engine, dihedrals, grid_spacing, init_coords_M=None, energy_d
         # filename for storing finished task result
         self.task_result_fname = 'dihedral_scanner_task_result.p'
         # threshold for determining the energy decrease
-        self.energy_decrease_thresh = energy_decrease_thresh
+        self.energy_decrease_thresh = energy_decrease_thresh if energy_decrease_thresh is not None else 1e-5
```

Comment:
`energy_decrease_thresh or 1.e-5` works here.

Reply:
@dgasmith Although it's unlikely but if the user wants `energy_decrease_thresh=0.0`, then `energy_decrease_thresh or 1.e-5` will return `1.e-5`, overwriting the user setting.